clang/unittests/Format/ConfigParseTest.cpp

```diff
@@ -161,7 +171,8 @@ TEST(ConfigParseTest, ParsesConfigurationBools) {
   CHECK_PARSE_BOOL(IndentAccessModifiers);
   CHECK_PARSE_BOOL(IndentCaseLabels);
   CHECK_PARSE_BOOL(IndentCaseBlocks);
-  CHECK_PARSE_BOOL(IndentGotoLabels);
+  CHECK_PARSE_ENUM(IndentGotoLabels,
```

Comment:
We have dedicated sections on how to check enum parsing.

Reply:
`3047 |   Style.IndentGotoLabels = false; `in `llvm-project/clang/unittests/Format/FormatTest.cpp
`
I missed this and it's shameful and I corrected it but now looking carefully at my local end.

Reply:
I want you to also test the actual usage of your option